lib/iris/coords.py

```diff
@@ -85,6 +86,8 @@ def _sort_key(defn):
                     defn.long_name is not None, defn.long_name,
                     defn.var_name is not None, defn.var_name,
                     defn.units is not None, defn.units,
+                    # Should attributes be included here?
+                    # Should bounds_are_climatological be included here?
```

Comment:
@trexfeathers What behavioural difference does it make if it is included?

Note that, this is Python2 specific, so this will be a non-issue come Python3 and `iris` 3.0.0 :tada:

Reply:
I struggled to follow the purpose of this section when talking about it with @pp-mo . Given it's just an uncertainty and doesn't break any tests, I'm in favour of leaving this until Iris 3 solves the issue anyway. I'll remove the comment